var/spack/repos/builtin/packages/py-udunits/package.py

```diff
@@ -0,0 +1,71 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+import os
+
+
+class PyUdunits(PythonPackage):
+    """The MetOffice cf_units Python interface to the UDUNITS-2 Library."""
+    homepage = "https://github.com/SciTools/cf_units"
+    url      = "https://github.com/SciTools/cf_units/archive/v1.1.3.tar.gz"
+
+    version('1.1.3', '8f3a159f43bc407e21b85a9cae04d903')
+
+    maintainers = ['citibeth']
+
+    depends_on('py-six', type=('build', 'run'))
+    depends_on('py-netcdf', type=('build', 'run'))
+    depends_on('udunits2')
+
+    # See: https://github.com/SciTools/cf_units/blob/master/cf_units/etc/site.cfg.template
+    # udunits2_path = /path/to/libudunits2.so
+    # udunits2_xml_path = /path/to/udunits2.xml
+    site_cfg_template = """[System]
+udunits2_path = %s
+udunits2_xml_path = %s
+"""
+
+    def install(self, spec, prefix):
+        setup_py('install', '--prefix=%s' % prefix)
+
+        cfg_templates = spack.llnl.utils.filesystem.find(
```

Comment:
`spack` hasn't been imported, has it? You shouldn't need to use the full path, `find()` should already be present thanks to `from spack import *`.